apps/pm-user/src/components/atoms/index.js

```diff
@@ -4,9 +4,10 @@ export { default as Card } from './card';
 export { default as Carousel } from './carousel';
 export { default as Checkbox } from './checkbox'
 export { default as DatePicker } from './datePicker'
-export { default as Form } from './form'
-export { default as Input } from './input';
+export { default as Form  } from "./form";
+export { default as FormWraper } from "./formWraper";
```

Comment:
Please rename this component to "FormWrapper"
Spelling.

Reply:
ok doing it